### PR TITLE
frontend: Redirect evy.wasm filename to via wasmImports map

### PR DESCRIPTION
**Pull request by juliaogris**
Redirect evy.wasm filename to via wasmImports map so that we cache-bust
evy.wasm in the same way we are doing for the js files in the importmap.

This is a preparatory PR for PR #255 to also include evy.wasm cache-busting.

### CODE REVIEW

File: frontend/play/evy.wasm
Comment (juliaogris) [inline frontend/play/evy.wasm:1]: woops, that got accidentally added a couple of PRs ago.

File: frontend/play/index.html [hunk 1]
```diff
@@ -12,8 +12,14 @@
...
     <script type="importmap"></script>
-    <script src="wasm_exec.js" defer></script>
-    <script src="index.js" type="module" defer></script>
+    <script>
+      const wasmImports = {
+        // Right is replaced for cache busting, e.g. "./module/evy.wasm": "./module/evy.11738cdd.wasm"
+        "./module/evy.wasm": "./module/evy.wasm",
+      }
+    </script>
+    <script src="wasm_exec.js" type="module"></script>
```
Comment (juliaogris) [inline frontend/play/index.html:21]: I removed the `defer` here - apparently that's redundant with `type="module"`. if you want I'll move it into a separate commit.

File: frontend/play/evy.wasm
Comment (juliaogris) [reply frontend/play/evy.wasm:1]: nope, this is just a "dangling" link. nothing bad happened here.

File: frontend/play/index.html (hunk 1 above)
Comment (juliaogris) [reply frontend/play/index.html]: ok, I've moved this into a separate commit.

**Review by camh- — APPROVED** [review]
:broccoli:

**Pull request merged**